renku/cli/init.py

```diff
@@ -75,15 +75,40 @@
     https://github.com/SwissDataScienceCenter/renku-project-template@master
     ... OK
 
-    INDEX  ID              DESCRIPTION
-    -----  --------------  -------------------------------------------------
-    1  python-minimal  Basic Python Project: The simplest Python-based[...]
-    2  R-minimal       Basic R Project: The simplest R-based renku proj[...]
+    INDEX ID             DESCRIPTION                PARAMETERS
+    ----- -------------- -------------------------- ----------------------
+    1     python-minimal Basic Python Project:[...] description: proj[...]
+    2     R-minimal      Basic R Project: The [...] description: proj[...]
 
     Please choose a template by typing the index:
 
-Updating an existing project
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Provide parameters
+~~~~~~~~~~~~~~~~~-
+
+Some templates require parameters to properly initialize a new project. You
+can check them by listing the templates ``--list-templates``.
+
+To provide parameters, use the ``--parameter`` option and provide each
+parameter using ``--parameter "param1"="value1"``.
```

Comment:
Do we need to also put `param1` in the quotes? It's not necessary when there is no space and I guess parameter names have no space in them.

Reply:
Not sure if quotes are really needed here for `param1`, I assumed there may be a (not so trivial) valid python variable name that, when passed through the cli, could be misinterpreted without the quotes -- if that is not the case, I can remove them.

Reply:
It's OK, keeping them doesn't hurt.